expression-tracker/lib/pit-client.rb

```diff
@@ -40,19 +128,7 @@ def initialize(type, id, config={}, &block)
 		@client.connect
 		log("MQTT Connected to #{@config[:host]}")
 	
-		if block
```

Comment:
Moved this block calling code to `subscribe` so I could call it myself after construction.